charts/handbrake/0.0.1/values.yaml

```diff
@@ -0,0 +1,87 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+  strategy:
+    type: Recreate
+
+  service:
+    port:
+      port: 5800            #Required for web GUI
+    additionalServices:
+      - enabled: true
+        nameSuffix: handbrake-vnc
+        type: ClusterIP
+        port:
+          port: 5900        #Optional, only for vnc clients.
+          name: handbrake-vnc
+          protocol: TCP
+          targetPort: 5900
+  
+
+  #All values here are set as the docker defaults.
+  env:
+  # Permissions Settings
+    USER_ID: "1000"
+    GROUP_ID: "1000"
+    SUP_GROUP_IDS:
+    UMASK:    
+  # General Settings
+    #TZ: Etc/UTC    #Do we need this?
+    KEEP_APP_RUNNING: "0"
+    APP_NICENESS: 
+    CLEAN_TMP_DIR: "1"
+  # GUI Settings
+    DISPLAY_WIDTH: "1280"
+    DISPLAY_HEIGHT: "768"
+    SECURE_CONNECTION: "0"
+    VNC_PASSWORD:
+    X11VNC_EXTRA_OPTS:
+    ENABLE_CJK_FONT: "0"
+  # Automated Conversion Preset
+    AUTOMATED_CONVERSION_PRESET: General/Very Fast 1080p30
+    AUTOMATED_CONVERSION_FORMAT: mp4
+    AUTOMATED_CONVERSION_KEEP_SOURCE: "1"
+    AUTOMATED_CONVERSION_VIDEO_FILE_EXTENSIONS:
+    AUTOMATED_CONVERSION_NON_VIDEO_FILE_ACTION: ignore
+    AUTOMATED_CONVERSION_NON_VIDEO_FILE_EXTENSIONS: jpg jpeg bmp png gif txt nfo
+    AUTOMATED_CONVERSION_OUTPUT_DIR: /output
+    AUTOMATED_CONVERSION_OUTPUT_SUBDIR:
+    AUTOMATED_CONVERSION_OVERWRITE_OUTPUT: "0"
+    AUTOMATED_CONVERSION_SOURCE_STABLE_TIME: "5"
+    AUTOMATED_CONVERSION_SOURCE_MIN_DURATION: "10"
+    AUTOMATED_CONVERSION_CHECK_INTERVAL: "5"
+    AUTOMATED_CONVERSION_MAX_WATCH_FOLDERS: "5"   ##If we let users set infinite watch folders, we need to update this dynamically or expose to UI to let them update it.
+    AUTOMATED_CONVERSION_NO_GUI_PROGRESS: "0"
+    AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS:
+    AUTOMATED_CONVERSION_INSTALL_PKGS:
+
+  persistence:
+    config:
+      enabled: false
+      emptyDir: false
+      mountPath: /config
+
+    storage:
+      enabled: false
+      emptyDir: false
+      mountPath: /storage  # This can be read only
+
+    watch:      
+      enabled: false
+      emptyDir: false
+      mountPath: /watch
+
+    output:
+      enabled: false
+      emptyDir: false
+      mountPath: /output
+
+
+## Somehow must enable user to add as many watch folders and env vars as he wants. 
+## All folders must start with "watch" and an index after
+## eg. /watch1, /watch2
```

Comment:
You can either create a watch folder, and mkdir: /watch/watch1 /watch/watch2 /watch/watch3 etc

OR

Wait till I implement it, its on my todo list to allow unlimted custom volumemounts ;-)

Reply:
This will wait then until implemented.
This container looks for /watch# 
So you can override settings with ENVVAR_# .
Can't override settings for subdirs!